whynotearth/BrowTricks
# 369 change place of form template starter

# Issue Being Addressed

#369 

# Type of PR

[ ] Bug Fix
[ ] Refactor
[ ] New Feature
[x] Update to Existing Feature
[ ] Other (state below)

# Description

**For Feature Updates:** What feature are we updating? Explain your technical approach to doing this.  
- Remove 'disabled navigation bottom'
- Remove PMU form template starter page from the initial steps of signup
- Show PMU form template starter page in clinet -> PMU Forms page (if there is no form template)

# How to Test/Reproduce

1. Signup a new user and create a tenant (it should route to tenant home after creating a tenant)
2. Create a client
3. Go to client page
4. Go to PMU Forms

It should show the starter page for form-templates.  
Next time, After creating the first form-template, in 'client -> PMU Forms' it should not show the starter page, just show the form templates list.

# Additional details

- Added **Edit Your Number** link to verify number page
